### PR TITLE
Updated demo for app-runtime secrets set by app developer or requester

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

**Review by mcornaton — APPROVED** [review]
Looks great!

File: cloud-computing/src/app.py [hunk 1]
```diff
@@ -60,25 +61,38 @@ def handle_input_files():
 
 def handle_app_provider_secrets():
```
Comment (zguesmi) [inline cloud-computing/src/app.py:62]:
```suggestion
def handle_app_developer_secrets():
```

File: cloud-computing/src/app.py [hunk 2]
```diff
@@ -97,10 +111,13 @@ def save_result(text):
...
     input_files_text = handle_input_files()
     print(input_files_text)
-    app_provider_secrets_text = handle_app_provider_secrets()
-    print(app_provider_secrets_text)
-    result = f'{computation_text}\n' \
-             f'{dataset_text}\n' \
-             f'{input_files_text}\n' \
-             f'{app_provider_secrets_text}\n'
-    save_result(result)
+    app_developer_secrets_text = handle_app_provider_secrets()
```
Comment (zguesmi) [inline cloud-computing/src/app.py:114]:
```suggestion
    app_developer_secrets_text = handle_app_developer_secrets()
```

File: cloud-computing/src/app.py [hunk 3]
```diff
@@ -97,10 +111,13 @@ def save_result(text):
...
+    print(app_developer_secrets_text)
+    requester_secrets_text = handle_requester_secrets()
+    print(requester_secrets_text)
+result = f'{computation_text}\n' \
```
Comment (zguesmi) [inline cloud-computing/src/app.py:118]:
```suggestion
    result = f'{computation_text}\n' \
```

File: cloud-computing/src/app.py (hunk 2 above)
Comment (jeremyjams) [reply cloud-computing/src/app.py]: Updated all, thanks!

**Review by zguesmi — APPROVED** [review]
Thanks

**Pull request merged**